test/acceptance/tests.py

```diff
@@ -1045,7 +1045,7 @@ def test_staff_override_at_beginning(self):
         self.staff_area_page.verify_learner_final_score(self.STAFF_OVERRIDE_STAFF_AREA_NOT_COMPLETE)
 
         # Do staff override
-        self.do_staff_override(learner, self.STAFF_OVERRIDE_STAFF_AREA_NOT_COMPLETE)
+        self.do_staff_override(learner)
```

Comment:
Both of the changes in this file are the same. I ought to have updated this method call in https://github.com/edx/edx-ora2/pull/931, but missed it due to a quirk of running tests locally (I imagine my devstack was slow enough that the previous `STAFF_OVERRIDE_STAFF_AREA_NOT_COMPLETE` string was visible longer than it ought to have, Jenkins was fast enough to expose this)